tests/test_requests.py

```diff
@@ -987,6 +988,15 @@ def test_response_decode_unicode(self):
         chunks = r.iter_content(decode_unicode=True)
         assert all(isinstance(chunk, str) for chunk in chunks)
 
+    def test_response_reason_unicode(self):
+        # check for unicode HTTP status
+        r = requests.Response()
+        r.url = six.u('unicode URL')
+        r.reason = six.u('Komponenttia ei löydy').encode('utf-8')
```

Comment:
As we discussed above, we don't support 3.0, 3.1, or 3.2. 2.7 absolutely supports `u''`. And you're using `u''` in your pull request inside requests.